browser/src/__tests__/NameForKey.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { configure} from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+import {expect} from 'chai'
+import Helper from '../util/NameForKey'
+
+configure({adapter: new Adapter()});
+
+describe("Map of class to Provider has correct short name", () => {
+
+    const nodeAutoSuggest : Helper = new Helper()
+  
+    test('test 1, from CoffeeApp Example', () => {
+      expect(nodeAutoSuggest.displayNameForKey("com.snap.dagger.browser.sample.CoffeeApp.CoffeeShop")).equals(
+        "CoffeeApp.CoffeeShop")
+    })
+
+    test('test 3, from HomeComponent', () => {
```

Comment:
`from HomeComponent` is not a good test name, maybe something like `displayNameForKey  should return simplified annotation and class name`. Not sure about the best practice for test naming for typescript, but you got what I mean.